docs/.vitepress/config.ts

```
@@ -1,24 +1,34 @@
 import type { DefaultTheme } from 'vitepress'
 import { transformerTwoslash } from '@shikijs/vitepress-twoslash'
 import { defineConfig } from 'vitepress'
-import { groupIconMdPlugin } from 'vitepress-plugin-group-icons'
 import { version } from '../../package.json'
+import llmstxt from 'vitepress-plugin-llms'
```

ESLint

[error] Expected "vitepress-plugin-llms" (external) to come before "../../package.json" (parent).

(perfectionist/sort-imports)